docker-compose.yml

```diff
@@ -11,6 +11,13 @@ services:
             - ./public/index.html:/usr/share/nginx/html/index.html
             - ./public/list.html:/usr/share/nginx/html/list.html
 
+    mdenet-tokenserver:
+        image: mdenet-tokenserver:latest
+        build:
+          context: educationplatform/tokenserver
+        ports:
+            - "10000:10000"
```

Comment:
The tokenserver cannot be made private to the container as initially discussed because the education platform runs in the user's browser which is external to the container. 

This is ok for local testing, for deployment configuring a ssl connection for the tokenserver is advisable.

Reply:
No but I'll document as part of https://github.com/mdenet/educationplatform/issues/69

Reply:
OK